bat-utils/lib/youtube.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const braveHapi = require('./extras-hapi')
+
+async function getYoutubeChannelId (youtubeUsername) {
+  const youtubeApiKey = process.env.YOUTUBE_API_KEY
+  const youtubeApiBaseUri = process.env.YOUTUBE_API_BASE_URI || 'https://www.googleapis.com'
```

Comment:
Should this be an envar? It is on publishers but I don't really see this changing.

Reply:
might as well keep it as such. quicker update times